app/controllers/structures_controller.rb

```diff
@@ -21,14 +21,18 @@ def show
   end
 
   def update
-    status = StructureUpdater.from_csv(@cocina, params[:csv].read)
-
-    if status.success?
```

Comment:
This syntax is much clearer to me. (if/then vs lambdas).

Reply:
I'm not attached. I'm going to merge to fix the error, but feel free to rejigger for readability.  FWIW, I this is the more idiomatic according to the dry-monad docs.

Reply:
Yeah, but every time I've looked at the dry-monad docs I've been unable to wrap my head around the idiom.